instrumentation/rake/test/opentelemetry/instrumentation/rake/instrumentation_test.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'test_helper'
+
+require_relative '../../../../lib/opentelemetry/instrumentation/rake'
```

Comment:
Would a better place to load these be in the test helper by `require 'opentelemetry-instrumentation-rake'` or using `Bundler.require`?

Reply:
this is from the instrumentation generator:
https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/1af9fc1a35264dcaf3bd0d88234e8ad8dacdaa22/.instrumentation_generator/templates/test/instrumentation.rb.tt#L9

and other instrumentations are doing the same...

but should I modify it?

Reply:
No worries. I will submit a separate PR to leverage bundler instead.